components/tabstrip/overview.md

```diff
@@ -12,20 +12,19 @@ position: 0
 
 This article provides information about the <a href = "https://www.telerik.com/blazor-ui/tabstrip" target = "_blank">Blazor TabStrip component</a> and its core features.
 
-#### Basics
+>caption In this article:
+* [Basics](#basics)
+* [Features](#features)
+* [Example](#example)
 
-The TabStrip is defined through the `<TelerikTabStrip>` tag that accepts children of type `<TabStripTab>`. Inside the tabs you can add content like in any other container, including other components.
+## Basics
 
-The tab exposes a `Title` attribute that is the text rendered in the heading. It also offers the `Disabled` attribute that allows you to disable its selection.
+To use a Telerik TabStrip for Blazor:
 
-The `ActiveTabIndex` parameter lets you get and set the currently shown tab index through two-way binding, and also provides an event for the tab change. To deactivate all tabs, set it to `-1`.
-
-To control the position of the tab titles, the main TabStrip tag exposes the optional `TabPosition` attribute that takes a member of the `Telerik.Blazor.TabPosition` enumeration:
-
-* `Top` (default)
-* `Left`
-* `Right`
-* `Bottom`
+1. Use the `<TelerikTabStrip>` tag
+1. Inside it add an instance of `<TabStripTab>` tag for each tab you want to include in the component
+1. Place the desired content in the tabs - it can be any content, including other components
+1. (Optional) Configure the separate tab instances and their position - Read more in [Tabs Configuration]({%slug tabstrip-tabs-configuration%}) and [Tabs Position]({%slug tabstrip-tabs-position%}) articles.
```

Comment:
This point is not easily actionable and does not provide a quick suggestion about a specific setting. I suggest that we directly instruct the reader to set `Title` - before the step about the tab content.

Reply:
Updated.